scbd/absch.cbd.int
# chm360 -fix search page issue

### General description
fix following issues:
1. Country filter shows json object
2. Country regions facets are not showing
3. My saved searches tab is showing translation keys
4.  remove homepage-> menu->search-> "national report analyzer" 
5. remove homepage->get start-> search->  "chmHomeT.registries "
6. remove homepage->menu->help->training materials


details:
1. Country filter shows json object
![image](https://github.com/scbd/absch.cbd.int/assets/65098066/3a020957-840a-46ff-bd6c-136cb1744ddd)

issue reason  : 
after import alphabets from '~/app-text/views/search/alphabets.json';   the alphabets value  contains {“en” in the begining of the object, so in the code, it treat the whole list as one item. 
how to fix: 
change alphabet to alphabet[locale] in html file

2.Country regions facets are not showing

issue reason:
when loading search page, it interrupted by error, because when calling loadLeftMenuFieldMapping function, it doesn’t contain CHM content. 
how to fix : 
add relevant code, and filter file and json file. 
after fix: 
<img width="943" alt="chm360-search result" src="https://github.com/scbd/absch.cbd.int/assets/65098066/c7e4d418-8863-4dd1-bdd4-dd392f289989">

3. My saved searches tab is showing translation keys

issue reason: 
when calling translation service, it is used inside the function, so it doesn’t work.
how to fix: 
move the translationService.set('userAlertsT', userAlertsT) outside the function ,put it under app.directive statement.
after fix: 
<img width="925" alt="chm360-saved" src="https://github.com/scbd/absch.cbd.int/assets/65098066/d700130b-7485-47ec-8125-c0b482fdab7a">



### Testing instructions
testing searchpage
1. in every tab, show correct records number 
2. in country tab, show A-Z, and has correct link
3. click saved search , show correct statesment not key.
4. check menu and getstart in homepage ,   
5. test ABS/BCH, ensure the change doesn't inflence other platform.
